zkevm-circuits/src/evm_circuit/util/math_gadget/constant_division.rs

```diff
@@ -60,7 +60,7 @@ impl<F: Field, const N_BYTES: usize> ConstantDivisionGadget<F, N_BYTES> {
         self.quotient.expr()
     }
 
-    pub(crate) fn remainder(&self) -> Expression<F> {
+    pub(crate) fn _remainder(&self) -> Expression<F> {
```

Comment:
Think it will be better by

```rust
#[allow(dead_code, reason = "reserve method for extract remainder expression")]
pub(crate) fn remainder(&self) -> Expression<F>
```